### ISSUE TITLE
feat: add ability to switch networks

### ISSUE THREAD

**Issue by sirpy** [GoodDollar/GoodCollective#214]
see also #181

If user is not connected to Celo currently UI just shows unsupported network.
Usually it should be a button or modal triggering switch network request to metamask.
Also since we are just on celo perhaps it should already say switch to celo

<img width="548" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMTNKQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8598059e034aeb11957c49d1c83c8d24232beea2/image.png" alt="image.png" />

Implementation Description:
When user is connected to a non supported network the UI should automatically ask user to switch network
- only trigger switch if user has connected a wallet
- show a modal to user asking to switch network
- trigger switch network on metamask like wallets

Technical description
- handle edge case when connecting to a wallet via walletconnect, and the wallet doesnt support Celo

**Comment by decentralauren** [comment]
@sirpy good catch. Let's please follow the pattern we have on GoodDapp / implementation we have there. cc @L03TJ3

**Comment by eben619** [comment]
Hello, @sirpy @decentralauren can i get assigned to this?

**Comment by sirpy** [comment]
@eben619 are you refering to the scouts game? I don't believe you need to be assigned to it, just submit a PR

**Comment by Immadominion** [comment]
Can I be assigned to this task? 

I can follow the implementatio @ https://github.com/L03TJ3 to add ability to switch networks

**Comment by sirpy** [comment]
@Immadominion @eben619 you don't need to be assigned, just propose a PR
